pkg/metrics/listener.go

```diff
@@ -30,7 +33,7 @@ var log = logf.RuntimeLog.WithName("metrics")
 var DefaultBindAddress = ":8080"
 
 // NewListener creates a new TCP listener bound to the given address.
-func NewListener(addr string) (net.Listener, error) {
+func NewListener(addr string, secureServing bool) (net.Listener, error) {
```

Comment:
Given that we're doing a breaking change here, should we convert to options?

Reply:
Done, broke even more while we're at it..